eclipse/sw360
# Fix(Component UI): Renaming component changes the linked release name

**Summary:** Renaming a component will rename the linked releases also.
**Steps to test:**
- Login to SW360 as `Admin`, and open `Component` portal
- Edit any Component (with and without release) listed under Component tab.
- Rename the component.
- Save the component.
- Once saved, check the component name is changed, and also the name of the linked releases is changed to same as component.

Do the same testing by logging in as `User` and also via REST Endpoint.

closes: #625

Signed-off-by: Abdul Kapti <abdul.mannankapti@siemens.com>
